src/app.rs

```diff
@@ -48,40 +48,117 @@ pub fn BlockchainExplorer() -> impl IntoView {
     });
 
     view! {
-        <div class="flex flex-col items-center">
-            <h1 class="mb-4 text-4xl font-extrabold leading-none tracking-tight text-gray-900 md:text-5xl lg:text-6xl">
-                Forest Explorer
-            </h1>
-            <select on:change=move |ev| { rpc_context.set(event_target_value(&ev)) }>
-                <option value=Provider::get_network_url(Network::Testnet)>Glif.io Calibnet</option>
-                <option value=Provider::get_network_url(Network::Mainnet)>Glif.io Mainnet</option>
-            </select>
-            <p>StateNetworkName</p>
-            <Transition fallback=move || view! { <p>Loading network name...</p> }>
-                <p class="px-8">
-                    <span>{move || network_name.get().flatten()}</span>
-                    <Loader loading=move || network_name.get().is_none() />
+        <main class="min-h-screen flex flex-col flex-grow space-y-8">
+            <div class="space-y-6 flex flex-col items-center">
+                <h1 class="text-4xl font-extrabold leading-none tracking-tight text-gray-900 md:text-5xl lg:text-6xl">
+                    Forest Explorer
+                </h1>
+                <p class="max-w-2xl">The Filecoin Forest Explorer Faucet provides developers and users with free testnet and mainnet FIL to support their exploration and development on the Filecoin network.
                 </p>
-            </Transition>
+            </div>
 
-            <p>StateNetworkVersion</p>
-            <Transition fallback=move || view! { <p>Loading network version...</p> }>
-                <p class="px-8">
-                    <span>{move || network_version.get().flatten()}</span>
-                    <Loader loading=move || network_version.get().is_none() />
-                </p>
-            </Transition>
-            <button class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-1 px-2 rounded-full">
+            <div class="grid grid-cols-1 md:grid-cols-2 gap-6 max-w-4xl w-full m-auto">
+                <div class="bg-white p-6 rounded-lg border border-gray-100">
+                    <h2 class="text-lg font-semibold text-gray-900 mb-4">What does the faucet offer?</h2>
+                    <ul class="space-y-3">
+                        <li class="flex items-start">
+                            <svg class="h-5 w-5 text-green-500 mr-2 flex-shrink-0" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 13l4 4L19 7" />
+                            </svg>
+                            <span class="text-gray-600">Free testnet FIL for experimentation and development.</span>
+                        </li>
+                        <li class="flex items-start">
+                            <svg class="h-5 w-5 text-green-500 mr-2 flex-shrink-0" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Comment:
Can we create separate components for the `CheckIcon` and `LightningIcon` SVGs and reuse them to reduce code repetition and improve maintainability?